packages/clarity-decode/src/data.ts

```diff
@@ -57,8 +57,7 @@ export function decode(tokens: Data.Token[]): DataEvent {
                 scrollY: tokens[8] as number,
                 pointerX: tokens[9] as number,
                 pointerY: tokens[10] as number,
-                activityTime: tokens[11] as number,
-                scrollTime: tokens.length > 12 ? tokens[12] as number: null
```

Comment:
this is problematic since it can cause exceptions when we add another type field in the same position in future